.config/devbox/scripts/dev-app-configure-and-start.sh

```diff
@@ -4,42 +4,35 @@ set -e
 (
   if [ ! -d "_dev-app" ]
   then
-    node ./packages/create-webstone-app/bin.js _dev-app --type=application
+    node ./packages/create-webstone-app/bin.js _dev-app --type=app
   fi
   cd _dev-app
   echo "⏳ Waiting for the CLI's binary to be built..."
   while [ ! -f ../packages/cli/dist/bin.js ]; do sleep 1; done
-  pnpm add -D ../packages/cli
+  npm install -D ../packages/cli
```

Comment:
Switching this to `npm` rather than `pnpm` so it doesn't pollute the root-level `pnpm-lock.yaml` file.

Even if we exclude the `_dev-app` package in `pnpm-workspace.yaml`, running `pnpm` commands updates the root-level `pnpm-lock.yaml`.